src/variables.py

```diff
@@ -41,6 +41,11 @@
     MELLOW_CONTRACT_ADDRESS = Web3.to_checksum_address(MELLOW_CONTRACT_ADDRESS)
 VAULT_DIRECT_DEPOSIT_THRESHOLD = Web3.to_wei(*os.getenv('VAULT_DIRECT_DEPOSIT_THRESHOLD', '1 ether').split(' '))
 
+# Withdrawal Queue proxy address
+# Mainnet: 0x889edC2eDab5f40e902b864aD4d7AdE8E412F9B1
+# Holesky: 0xc7cc160b58F8Bb0baC94b80847E2CF2800565C50
+WITHDRAWAL_QUEUE_PROXY_ADDRESS = os.getenv('WITHDRAWAL_QUEUE_PROXY_ADDRESS', '0x889edC2eDab5f40e902b864aD4d7AdE8E412F9B1')
```

Comment:
Withdrawal queue address could be found in lido_locator:
https://etherscan.io/address/0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb#readProxyContract

Reply:
Done